docs/getting-into-the-code/guidelines-for-setting-up-a-local-workspace.md

```diff
@@ -142,13 +143,20 @@ After that a new entry called "jabref \[run\]" will appear in the run configurat
 
 You should use IntelliJ IDEA's internal build system for compiling and running JabRef during development, because it is usually more responsive. Thereby, **it's important** that you understand that JabRef relies on generated sources which are only build through Gradle. Therefore, to build or update these dependencies you need to run the `assemble` Gradle task at least once.
 
+Note that these steps do not work on IntelliJ 2020.2.
+
 To use IntelliJ IDEA's internal build system when you build JabRef through **Build \| Build Project** or use the provided "JabRef Main" run configuration, ensure that
 
 * In **File \| Settings \| Build, Execution, Deployment \| Build Tools \| Gradle** the setting "Build and run using" and "Test using" is set to "IntelliJ IDEA".
-
 * Ignore the Gradle project "buildSrc" by clicking the button **Select Project Data To Import** in the Gradle Tool Window and unchecking the folder "buildSrc".  
-
    ![Ignore the Gradle project &quot;buildSrc&quot;](../.gitbook/assets/intellij-gradle-config-ignore-buildSrc.png)
+* Add the folder `src/main/gen` as generated sources root  
+   ![src/main/gen is generated](../.gitbook/assets/intellij-mark-src-main-gen-as-generated-sources-root.png.png)
+* Delete `org.jabref.gui.logging.plugins.Log4jPlugins`. Otherwise, you will see folowing error:
```

Comment:
This step should only be necessary for developers that used the old version (where the log4jplugin was created in a different folder).

Reply:
After `gradlew run`, the file is generated. I checked it with latest master. Thus, the file always has to be deleted.

Reply:
Yes, it should be generated in "src/main/gen" because of https://github.com/JabRef/jabref/blob/master/build.gradle#L376. But that's exactly what it's designed to do. So I'm not sure why you would want to remove this file. It's needed to build.

Reply:
When removing it, the internal IntelliJ compiler perfectly compiles. When it is present, IntelliJ 2020.1 fails with the error.

Is it different on your side?

(I am not talking about the gradle build. I aim for offering all available build chains (IntelliJ build-in and gradle) to the users)